Whollet/Whollet/Source/Presentation/Wallet/WalletViewController.swift

```diff
@@ -0,0 +1,117 @@
+                            self.nextButton.backgroundColor = UIColor.MyTheme.primary.withAlphaComponent(0.5)
+                        }
+                    }
+                case .loadSuccess:
+                    if self.initialState == .create {
+                        DispatchQueue.main.async {
+                            self.descriptionText.text = AppConstants.Strings.createWalletSuccessfully
+                            self.nextButton.setTitle(AppConstants.Strings.goHome, for: .normal)
+                            self.nextButton.backgroundColor = UIColor.MyTheme.primary
+                        }
+                    } else {
+                        DispatchQueue.main.async {
+                            self.descriptionText.text = AppConstants.Strings.loadWalletSuccessfully
+                            self.nextButton.setTitle(AppConstants.Strings.goHome, for: .normal)
+                            self.nextButton.backgroundColor = UIColor.MyTheme.primary
+                        }
+                    }
+                default:
+                    break
+                }
+            }
+        }
+    }
+    
+    private func configView() {
+        nextButton.fullCornerRadiusWithHeight()
+        nextButton.resizeTextWithHeight()
+        guard let initialState = initialState else {
+            return
+        }
+        switch initialState {
+        case .create:
+            DispatchQueue.main.async { [weak self] in
+                guard let self = self else { return }
+                self.navigationItem.title = AppConstants.Strings.createWallet
+                DispatchQueue.main.async {
+                    self.descriptionText.text = AppConstants.Strings.createICX
+                    self.nextButton.setTitle(AppConstants.Strings.createWallet, for: .normal)
+                    self.nextButton.backgroundColor = UIColor.MyTheme.primary
+                }
+            }
+        case .load:
+            DispatchQueue.main.async { [weak self] in
+                guard let self = self else { return }
+                self.navigationItem.title = AppConstants.Strings.loadWallet
+                DispatchQueue.main.async {
+                    self.descriptionText.text = AppConstants.Strings.copyPrimaryKey
+                    self.nextButton.setTitle(AppConstants.Strings.pasteKey, for: .normal)
+                    self.nextButton.backgroundColor = UIColor.MyTheme.primary
+                }
+            }
+        }
+    }
+    
+    @IBAction func nextButtonOnClick(_ sender: UIButton) {
+        guard let initialState = initialState else {
+            return
+        }
+        
+        switch viewModel.bsWalletState.value {
+        case .initial:
+            if initialState == .create {
+                viewModel.createWallet()
+            } else {
+                viewModel.loadWallet(UIPasteboard.general.string ?? "")
+            }
```

Comment:
bỏ return ở các case
```suggestion
            initialState == .create ? viewModel.createWallet() :  viewModel.loadWallet(UIPasteboard.general.string ?? "")
```